src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
Since unit tests pass whatever `auto + series` or `auto + linear`, could be great to have you guys testing use cases that could break. I didn't succeed to have different results between both combinations, even with explicit min/max.